### DISCUSSION TITLE
Some performance improvements

### DISCUSSION THREAD

**Discussion by stereobooster** [jpanther/congo#644] [Ideas]
1. [x] Add [quicklink](https://getquick.link/). There are alternative solutions like [swup](https://swup.js.org/), but quicklink seems to be a safe choice here
2. [x] [Add `width` and `height` to images](https://web.dev/optimize-cls/#images-without-dimensions)
3. [ ] Maybe add [lqip](https://github.com/transitive-bullshit/lqip-modern) to images (but this is matter of taste)
4. [x] ~~I wonder if Hugo has something like https://rviscomi.github.io/capo.js/~~
5. [x] [Maybe add width and height to SVG](https://github.com/jpanther/congo/pull/693)

**Comment by stereobooster** [comment]
RE lqip. I found two modes for this:

- https://github.com/hugo-mods/lazyimg
- https://github.com/danielfdickinson/image-handling-mod-hugo-dfd

But I would prefer to re-implement it. [I did it before](https://dev.to/stereobooster/responsive-images-for-hugo-dn9)

**Comment by jpanther** [comment]
Okay wow, thanks for all these proposed changes! Lots to digest here but I appreciate that you've submitted them as smaller PRs that can be dealt with individually. To keep everything succinct, I'll go through and provide comments as necessary on the relevant PR.

**Reply by stereobooster** [reply]
Not my first time 😄 . Thank you for creating such a great theme. I really like it

**Comment by stereobooster** [comment]
One more idea. Can we somehow get width and height for svg images? (Like we do for raster images) 🤔

**Reply by stereobooster** [reply]
https://github.com/gohugoio/hugo/issues/3700#issuecomment-705857100
